README.md

```diff
@@ -30,27 +30,37 @@ Our goal is to allow event organizers to set up professional conference website
 
 ## Getting Started
 1. [Fork repository](https://github.com/gdg-x/hoverboard/fork) and clone it locally
-1. Setup Environment
+2. Setup Environment
```

Comment:
Numérotation in Markdown doesn't work like that ;)

All the elements in a numbered list share the same number.

I.e.

```
1. Hello
1. world
```
Donne : 
1. Hello
1. world

Reply:
You can also use different values for the chips (according to https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet, "Lists" section) :)